terraform/groups/ecs-service/main.tf

```diff
@@ -0,0 +1,85 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+terraform {
+  backend "s3" {
+  }
+  required_version = ">= 1.3, < 2.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+module "secrets" {
+  source      = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.293"
```

Comment:
there is a double slash in the source paths - terraform-modules//aws - is this correct?

Reply:
@markpit Yes